### ISSUE TITLE
Allow to install apps from the app store with the provisoning API

### ISSUE THREAD

**Issue by schiessle** [nextcloud/server#20499]
If I have access to the server I can already use the occ command "app:install" to install an app from the app store but with the provisioning API I can only enable/disable already existing apps.

Sometimes it is easier to automate stuff from the outside over http calls, especially If you want to maintain multiple Nextclouds from one central place. Therefore it would be nice if the installation of apps from the appstore would be exposed over the provisioning API.

**Comment by ostasevych** [comment]
I also suggest to add the possibility to save the list of apps for quick roll-out on another instance.

**Comment by szaimen** [comment]
Is this Issue still valid? If not, please close this issue. Thanks! :)

**Comment by ostasevych** [comment]
In my learning CI project I am provisioning using CSV file by parsing each row with occ:install, occ:enable, occ:disable; occ:delete sequential operations. The list I've got just by running ls in a fresh installation. What I found that some apps I cannot disable, others I cannot delete, and some others may have additional options, which I may provide as well, eg, URL for documentserver_community app. It may be a good idea to maintain this list by nextcloud itself.

Attaching my example of the provisioning file.
[apps.csv](https://github.com/nextcloud/server/files/6816453/apps.csv)
